pkg/network/ovn_kubernetes.go

```diff
@@ -789,6 +792,27 @@ func getNodeListByLabel(kubeClient cnoclient.Client, label string) ([]string, er
 	return nodeNames, nil
 }
 
+// getKeyAndValueFromLabel returns specified label key and value (if any).
+// label format: "key=" (match any value, use operator: Exists) or "key=value" (match specific value, use operator: In)
+func getKeyValueFromLabel(label string) (string, []string, error) {
```

Comment:
Is the expectation here that when value is supplied it will only be a single value? If so, then the return type for value can be changed to `string` and go template for the `ovnkube-node.yaml` files can be updated accordingly. Otherwise, this function should parse individual values and then those should be used to create the string slice rather than `labelValues = []string{value}`.

Reply:
It should be a single string. You make a good point. If there is a need for multiple values then the config map should take a list of labels per field.